website/docs/docs/build/data-tests.md

```diff
@@ -13,6 +13,16 @@ keywords:
 * [Data test configurations](/reference/data-test-configs)
 * [Test selection examples](/reference/node-selection/test-selection-examples)
 
+<VersionBlock firstVersion="1.8">
+
+:::important
+
+With the addition of unit tests in dbt v1.8, what was previously called "tests" are now "data tests". In dbt v1.8, `tests:` is still supported in your YML configuration file as an alias for data tests but will be deprecated in the future in favor of `data_tests:`. See [New syntax](#new-tests-syntax) for more information.
```

Comment:
sugg: slightly more concise and improved readability 

```suggestion
In dbt v1.8, what was previously known as "tests" are now called "data tests". The YAML key `tests:` is still supported as an alias for data tests but will be deprecated in the future in favor of `data_tests:`. Refer to [New syntax](#new-tests-syntax) for more information.
```

Reply:
I think it's important to call out that this is the result of a new test type. How about:
```suggestion
In dbt v1.8, what was previously known as "tests" are now called "data tests" with the addition of [unit tests](/docs/build/unit-tests). The YAML key `tests:` is still supported as an alias for data tests but will be deprecated in the future in favor of `data_tests:`. Refer to [New syntax](#new-tests-syntax) for more information.
```

Reply:
love it!